### PR TITLE
fix: rollup minify selectors should retain spaces

### PR DESCRIPTION
**Pull request by radium-v**
# Description
Reduce complexity for CSS tagged template compression. Also spotted some invalid commas in the `progress` stylesheets.

## Motivation & context
The CSS tagged template minifier was combining `:host() .child` selectors, causing styles to not get rendered properly.

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/web-components/fast-components-msft/src/progress/progress/progress.styles.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export const ProgressStyles = css`
         height: 100%;
         background-color: ${accentForegroundRestBehavior.var};
         border-radius: calc(var(--design-unit) * 1px);
-        animation-timing-function: cubic-bezier(0.4, 0.0, 0.6, 1.0);
```
Comment (radium-v) [inline packages/web-components/fast-components-msft/src/progress/progress/progress.styles.ts:51]: These were likely automatically applied by vscode eslint.

File: packages/web-components/fast-components/rollup.config.js [hunk 2]
```diff
@@ -33,13 +37,20 @@ export default [
...
+            }),
             transformTaggedTemplate({
                 tagsToProcess: ["html"],
                 transformer: transformHTMLFragment,
                 parserOptions,
             }),
-            filesize({ showMinifiedSize: false }),
+            filesize({
+                showMinifiedSize: false,
+                showBrotliSize: true,
```
Comment (EisenbergEffect) [inline packages/web-components/fast-components/rollup.config.js:52]: Nice!

File: packages/web-components/fast-components-msft/src/slider/slider.styles.ts [hunk 3]
```diff
@@ -17,10 +17,6 @@ import {
 } from "../styles";
 
 export const SliderStyles = css`
-    :host([hidden]) {
-        display: none;
-    }
```
Comment (EisenbergEffect) [inline packages/web-components/fast-components-msft/src/slider/slider.styles.ts:22]: Was this removal intended?

Comment (radium-v) [reply packages/web-components/fast-components-msft/src/slider/slider.styles.ts:22]: Yes, the `${display("inline-grid")}` below already handles the `hidden` selector, right?

Comment (EisenbergEffect) [reply packages/web-components/fast-components-msft/src/slider/slider.styles.ts:22]: Would you look at that 🤣 I did not know that. Thanks for the attention to detail!
